### ISSUE TITLE
How are you *supposed* to configure e-mail? Because it's not working as expected.

### ISSUE THREAD

**Issue by walt93** [WWBN/AVideo#7173]
**Describe the question**
1. I thought, for some reason, that it would be a good idea to set my system email to "noreply@conspyre.tv" because - when I send out email - I do NOT want 1,000 users to **have** my e-mail. Not what we want to do, not at all, at all...

2. But, having my e-mail set to "noreply@" caused the site to break. With that e-mail, the contact form is broken.

3. So, silly me, I set my site e-mail back to my email address, frankly not sure why I should need to, but, after doing so - the contact form started working.

4.  So I just sent the monthly e-mail to my user base. **And immediately got replies to this e-mail because they sent with a from:of my legit email address**

<Big Sad Emoji>

```
X-Mailgun-Batch-Id: 6330f611cbe7639bc8259471
Sender: admin=conspyre.tv@mg.conspyre.tv
Date: Sun, 25 Sep 2022 17:44:58 -0700
To: undisclosed-recipients:;
From: [REDACTED HERE ON GITHUB]@conspyre.tv
Subject: =?UTF-8?Q?Message_From_Site_Conspyre=E2=80=A2TV?=
Message-ID: <nIwQgMfraMOZth5lYE2faFSSM57yuysVtDSP4K8iYTE@conspyre.tv>
X-Mailer: PHPMailer 6.6.4 (https://github.com/PHPMailer/PHPMailer)
MIME-Version: 1.0
Content-Type: multipart/alternative;
 boundary="b1_nIwQgMfraMOZth5lYE2faFSSM57yuysVtDSP4K8iYTE"
Content-Transfer-Encoding: 8bit

```

**Comment by walt93** [comment]
TL;dr I want to send email ALWAYS with a noreply@ email address
I also want the site's "contact" form to work.

How to achieve both?

**Comment by DanielnetoDotCom** [comment]
What do you mean your form is broken? please screenshot

**Comment by walt93** [comment]
1) Set up like this

![Screen Shot 2022-09-26 at 1 37 32 PM](https://user-images.githubusercontent.com/11711050/192375240-99b731bf-bc03-47db-93a0-35a8b0c78275.png)

2) Go here and fill out this form

![Screen Shot 2022-09-26 at 1 38 05 PM](https://user-images.githubusercontent.com/11711050/192375295-7312ea96-3dce-4a19-ac39-49baa05e9240.png)

3) No e-mail is received (e.g. contact form is broken; does not perform its intended function)

4) Put my valid e-mail address in here

![Screen Shot 2022-09-26 at 1 39 07 PM](https://user-images.githubusercontent.com/11711050/192375436-5d826421-c0f0-48e7-95d2-62424f724872.png)

5) Contact form now works

6) Now, go send the users an email

![Screen Shot 2022-09-26 at 1 40 36 PM](https://user-images.githubusercontent.com/11711050/192375705-47dab6d4-8cec-4f8c-bd77-137b5bf5c9d8.png)

7) ALL of the users now have myvalidemail@conspyre.tv which is something I ABSOLUTELY DID NOT WANT TO DO.

8) If I set the email address to "noreply@conspyre.tv" at least I can send e-mails to the users without them getting my valid e-mail address. But I break the contact form by doing so.

So I ask, how am I supposed to set this up. 1) I want the contact form to work 2) I don't want my e-mail to be sent in mass mailings.

I do not see how to achieve this.

**Comment by DanielnetoDotCom** [comment]
> If I set the email address to "[noreply@conspyre.tv](mailto:noreply@conspyre.tv)" at least I can send e-mails to the users without them getting my valid e-mail address. But I break the contact form by doing so.

I still do not know when you mean `"But I break the contact form by doing so."`

**Comment by walt93** [comment]
When the user fills out the contact form, no e-mail is received.

**Comment by DanielnetoDotCom** [comment]
If it is not working with "[noreply@conspyre.tv](mailto:noreply@conspyre.tv)" the problem is on your email provider, not on the form or on the code

you should contact your email provider to try to figure out why.

**Comment by walt93** [comment]
Noted and continuing to look for a solution. For the record this is with Mailgun, so if it's something strange it could be with them.

Are there any e-mail providers that this is known to work well with?

**Comment by DanielnetoDotCom** [comment]
I like to use Gmail for free

and I use this https://forwardemail.net/en/faq?domain=mydomain.com#how-do-i-get-started-and-set-up-email-forwarding

https://github.com/WWBN/AVideo/wiki/Setting-up-AVideo-Platform-to-send-emails#set-up-email-forwarding

**Comment by walt93** [comment]
Thanks Daniel!
